python/tskit/tables.py

```diff
@@ -3467,26 +3467,31 @@ def find_ibd(self, samples, max_time=None, min_length=None):
         segments returned for a given pair will partition the span of the contig
         represented by the tree sequence.
 
+        Each item in the outputted dictionary object contains the IBD segments for a
+        particular sample pair. Suppose the output is
```

Comment:
I don't think this will render well in rst (the backticks is Markdown syntax afaik) - have you looked at the output?